system-test/testnet-automation.sh

```diff
@@ -121,9 +124,17 @@ function launch_testnet() {
 
   execution_step "Starting bootstrap node and ${NUMBER_OF_VALIDATOR_NODES} validator nodes"
   if [[ -n $CHANNEL ]]; then
+    if [[ -z $NDEBUG ]]; then
+      maybeDebug=--debug
+    fi
+    # Our build machne is on Azure which supports AVX512 but colos don't support AVX512.
+    # So, turn off simd at all as a work around to correctly cross-compile
```

Comment:
I need to create a tiny build-related PR against solana-reed-solomon to remove this work-around.

Reply:
@mvines I just created!: https://github.com/solana-labs/reed-solomon-erasure/pull/1